### DISCUSSION TITLE
Is anybody using the Zig extension, and if yes is 'autofix' working for you?

### DISCUSSION THREAD

**Discussion by name-changer** [zed-industries/zed#17130] [Feature Requests]
Hi,

I use the Zig extension in Zed, and it basically works fine (completions, inline hints). But one feature doesn't work: the Zig LSP has a feature called 'autofix', that automatically discards unused variables by adding `_ = unused_var; //autofix`. 

This works for me in Helix and VSCode, but not in Zed. Zed only shows a⚡symbol next to the line, indicating I can discard the value manually, but it doesn't insert the discard automatically. 

Before I open an issue for this: does this work for anybody here? 

Thanks!

**Comment by name-changer** [comment]
_> but it doesn't insert the discard automatically._

Let me clarify: in Helix and VSCode the LSP automatically inserts the necessary discards when I save the file. That doesn't happen in Zed.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
